src_c/controllercompat.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file should not exist. If you exist in the future and are wondering if
+ * this file is cool, it is not.
+ */
```

Comment:
Some elaboration would help someone existing in the future to fix this 😉 

I think we need to clarify here why this file is being used, instead of `pgcompat.c`